src/NuGet.Frameworks/FallbackFramework.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Frameworks;
+
+namespace NuGet.Frameworks
+{
+    public class FallbackFramework : NuGetFramework, IEquatable<FallbackFramework>
+    {
+        /// <summary>
+        /// Secondary framework to fall back to.
+        /// </summary>
+        public NuGetFramework Fallback { get; }
+
+        public FallbackFramework(NuGetFramework framework, NuGetFramework fallbackFramework)
+            : base(framework)
+        {
+            Fallback = fallbackFramework;
```

Comment:
Could you add a arg null check here for `fallbackFramework`? We should never be calling this with a null value so let's codify it.